openslides/agenda/projector.py

```diff
@@ -193,12 +178,69 @@ def current_list_of_speakers_slide(
         item_id = model["agenda_item_id"]
         break
 
+    return item_id
+
+
+def get_reference_projector(all_data: AllData, projector_id: int) -> Dict[str, Any]:
+    """
+    Returns the reference projector to the given projector (by id)
+    """
+    try:
+        this_projector = all_data["core/projector"][projector_id]
+    except KeyError:
+        raise ProjectorElementException(f"Projector {projector_id} does not exist")
+
+    reference_projector_id = this_projector["reference_projector_id"] or projector_id
+    try:
+        reference_projector = all_data["core/projector"][reference_projector_id]
+    except KeyError:
+        raise ProjectorElementException(
+            f"Projector {reference_projector_id} does not exist"
+        )
+
+    return reference_projector
+
+
+def current_list_of_speakers_slide(
+    all_data: AllData, element: Dict[str, Any], projector_id: int
+) -> Dict[str, Any]:
+    """
+    The current list of speakers slide. Creates the data for the given projector.
+    """
+    reference_projector = get_reference_projector(all_data, projector_id)
+    item_id = get_current_item_id_for_projector(all_data, reference_projector)
     if item_id is None:  # no element found
         return {}
 
     return get_list_of_speakers_slide_data(all_data, item_id)
 
 
+def current_speaker_chyron_slide(
```

Comment:
That is "chyron"?

Reply:
A "Bauchbinde". The thing on e.g. news tv where the speaker and news are shown. I'm not responsible for this name ;)